api/src/wfl/api/handlers.clj

```diff
@@ -9,7 +9,7 @@
             [wfl.log                        :as log]
             [wfl.module.aou                 :as aou]
             [wfl.module.copyfile]
-            [wfl.module.covid]
+            [wfl.module.staged]
```

Comment:
Not sure why we need these now.
I bet this compiles without them.

Reply:
I've wondered this too.  I remember removing them in the past and Something Bad happened, I'll try to reproduce / be more specific.

Reply:
Looks like forgetting to import these modules previously caused some system tests to fail.

https://github.com/broadinstitute/wfl/pull/254

But I removed them and all system tests passed:

```
wm111-e35:wfl okotsopo$ make TARGET=system
export CPCACHE=/Users/okotsopo/wfl/api/.cpcache;            \
	export WFL_WFL_URL=http://localhost:3000; \
	clojure  -M:parallel-test wfl.system.v1-endpoint-test | \
	tee /Users/okotsopo/wfl/derived/api/system.log
WARNING: Specified path is external to project: ../derived/api/src
WARNING: Specified path is external to project: ../derived/api/resources

Ran 33 tests containing 374 assertions.
0 failures, 0 errors.
api system finished on Wed Feb 23 12:03:51 EST 2022
docs system finished on Wed Feb 23 12:03:51 EST 2022
functions/aou system finished on Wed Feb 23 12:03:51 EST 2022
functions/sg system finished on Wed Feb 23 12:03:51 EST 2022
helm system finished on Wed Feb 23 12:03:51 EST 2022
ui system finished on Wed Feb 23 12:03:51 EST 2022
```

Not sure what changed here… what do you think, safe to remove these in this PR?

Reply:
I'll remove.